v6/v6_interface.ml

```diff
@@ -20,17 +20,22 @@ let uri () = "file:" ^ !default_path
 
 type debug_info = string
 
+type edition_info = {
+    edition: string;
+    licence_restrictions: (string * string) list;
```

Comment:
This actually contains more than just the restrictions. There are other parameters as well, that end up in `host.license_params`. Let's call it `xapi_params`, like in the old code comment. And one below can be `additional_params`.